ext/standard/tests/strings/str_ends_with.phpt

```diff
@@ -10,15 +10,15 @@ var_dump(str_ends_with($testStr, "End"));
 var_dump(str_ends_with($testStr, "end"));
 var_dump(str_ends_with($testStr, "en"));
 var_dump(str_ends_with($testStr, $testStr."a"));
+var_dump(str_ends_with("a".$testStr, $testStr));
```

Comment:
Sorry, that's not what I had in mind (like this it's basically testing the same thing as the very first test), I meant `($testStr, "a".$testStr)` (and expect `false`).
Moreover, if you want to also keep `($testStr, $testStr."a")` here, then you should add `($testStr, "a".$testStr)` in the other file too ;)

Reply:
Haha I took it the way you intended it at first, then I told myself you might've meant the other way xD